tokenizers/src/pre_tokenizers/unicode_scripts/pre_tokenizer.rs

```diff
@@ -0,0 +1,100 @@
+use serde::{Deserialize, Serialize};
+
+use crate::pre_tokenizers::unicode_scripts::scripts::{get_script, Script};
+use crate::tokenizer::{normalizer::Range, PreTokenizedString, PreTokenizer, Result};
+
+#[derive(Serialize, Deserialize, Clone, Debug)]
+#[serde(tag = "type")]
+pub struct UnicodeScripts {}
```

Comment:
Did you check that the serialization works as expected with an empty C-Style struct? (cf https://github.com/huggingface/tokenizers/blob/master/tokenizers/src/pre_tokenizers/whitespace.rs where we `impl Deserialize` manually, and it seems similar). Maybe we want to use a unit-struct for consistency reasons if we don't have any field here.

Reply:
Will do